mlflow/server/handlers.py

```diff
@@ -2592,7 +2594,7 @@ def _create_logged_model():
 
     model = _get_tracking_store().create_logged_model(
         experiment_id=request_message.experiment_id,
-        name=request_message.name,
+        name=request_message.name or None,
```

Comment:
When `name` is not present in the request, that would default to an empty string, which isn't allowed.